modules/videoio/cmake/init.cmake

```diff
@@ -12,8 +12,13 @@ function(ocv_add_external_target name inc link def)
   set_target_properties(ocv.3rdparty.${name} PROPERTIES
     INTERFACE_INCLUDE_DIRECTORIES "${inc}"
     INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${inc}"
-    INTERFACE_LINK_LIBRARIES "${link}"
     INTERFACE_COMPILE_DEFINITIONS "${def}")
+  if (CMAKE_VERSION VERSION_LESS 3.11)
```

Comment:
Please add a brief comment with a link to this PR.

Reply:
Done.